Modules/binascii.c

```diff
@@ -1239,13 +1239,101 @@ binascii_b2a_qp_impl(PyObject *module, Py_buffer *data, int quotetabs,
     return rv;
 }
 
+/*[clinic input]
+binascii.b2a_base85
```

Comment:
It would feel weird not to have `binascii.a2b_base85` so I would suggest keeping it private for now. Ideally, `base64` should have its own C accelerator module but maybe it's an overkill.

Reply:
Renamed to private

That was what delayed me initially because I had no idea how to add a module dedicated to base64, I found out that it did use the binascii one only last week